src/components/NationalText/utils.tsx

```diff
@@ -1,55 +1,31 @@
 import React, { Fragment } from 'react';
-import AggregationsJSON from 'assets/data/aggregations.json';
-import last from 'lodash/last';
+import SiteSummaryJSON from 'assets/data/site-summary.json';
 import isNull from 'lodash/isNull';
-import {
-  formatPercent,
-  formatDecimal,
-  getPercentChange,
-  formatEstimate,
-} from 'common/utils';
+import { formatPercent, formatDecimal, formatEstimate } from 'common/utils';
 import {
   parseDateUnix,
   DateFormat,
   formatUTCDateTime,
 } from 'common/utils/time-utils';
 
-const getTwoWeekPercentChange = (series: any[]): number | null => {
-  const lastTwoWeeks = series.slice(-15);
-  const firstVal = lastTwoWeeks[0];
-  const lastVal = last(lastTwoWeeks);
-  if (!firstVal || !lastVal) {
-    return null;
-  }
-  return getPercentChange(firstVal, lastVal);
-};
-
 const getTotalCasesCopy = (summedRawCases: number): string => {
   const ONE_MILLION = 1000000;
   const divided = summedRawCases / ONE_MILLION;
   return `${formatDecimal(divided, 1)} million`;
 };
 
 export function getNationalText(): React.ReactElement {
-  const usaAggregation = AggregationsJSON['00001'];
+  const usa = SiteSummaryJSON.usa;
   const {
     totalCases,
     totalDeaths,
-    smoothedDailyCases,
-    smoothedDailyDeaths,
-    dates,
-  } = usaAggregation;
-
-  const percentChangeSmoothedCases = getTwoWeekPercentChange(
-    smoothedDailyCases,
-  );
-  const percentChangeSmoothedDeaths = getTwoWeekPercentChange(
-    smoothedDailyDeaths,
-  );
+    lastDate,
+    twoWeekPercentChangeInCases,
+    twoWeekPercentChangeInDeaths,
+  } = usa;
 
-  const lastDate = last(dates);
   const lastDateFormatted: string = formatUTCDateTime(
-    parseDateUnix(lastDate!),
+    parseDateUnix(lastDate),
```

Comment:
haha what did `lastDate!` compile to before?? that wasn't a syntax error!?

Reply:
Hah, `!` is the "[non-null assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator)" operator in TypeScript.  It tells TypeScript that even though the type says the value may be null you pinky-promise-swear that it won't actually be null and so it should let you pretend it won't ever be null.  It's similar to `any` in that whenever you use it, you're overriding TypeScript's type-checking and giving yourself the opportunity to shoot yourself in the foot.

In this case, since `lastDate` is now coming from the JSON file at compile-time, it can't be null (or rather TypeScript will fail the build if we ever screw up and put null in the JSON file), so `!` isn't necessary.

Reply:
ahh gotcha - TIL - thanks! it's been far too long since i've used TypeScript (4 years?) - i don't remember using that back in the day (instead i remember just marking things as `any`)